random-string-step/index.js

```diff
@@ -20,17 +20,16 @@ export const registerRandomStringStep = function (app) {
         strings: get(inputs, "strings.value", []),
       };
 
-      app.logger.info("Opening config view", state);
-      await app.client.views.open({
-        token: context.botToken,
+      logger.info("Opening config view", state);
+      await client.views.open({
```

Comment:
As `client` in listener args embodies `context.botToken`, you can simplify the code with it.

Reply:
interesting, I did not know that :)